LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright 2016 Yahoo Inc.
```

Comment:
2019?

Reply:
2016 is when we open sourced it. All our existing licenses reflect the same year.

Reply:
hm I think usually the license year matches when the repo was created
but I don't know what the rules are
@gyehuda ?

Reply:
I like Yahoo